locales/de/pdk.po

```diff
@@ -1,604 +0,0 @@
-# German translations for puppet development kit package.
```

Comment:
I have added a Rake task that cleans the file for us, one extra step to make sure it works, but we can keep the German file now